igvm/settings.py

```diff
@@ -162,6 +170,19 @@
     {'hypervisor': HYPERVISOR_ATTRIBUTES},
 ]
 
+AWS_DATACENTERS = [
+    'aws-fra',
```

Comment:
Wouldn't it be better to make this an attribute of a network in Serveradmin?

Reply:
Shouldn't it be handled like every other datacenter? Then it should stick to the VM

Reply:
Attribute of VM can be inherited (Serveradmin attribute inheritance) from network, that is not a problem. I only say that this should be a property in Serveradmin, not a hardcoded thing in IGVM.

Reply:
I removed `AWS_DATACENTERS` settings and introduced `igvm_operation_mode` attribute in SA with this regex `\A(aws|azure|esxi|hyperv|kvm)\Z`
I also put `kvm` (ig-chroot) as an elif in the common functions, for adding more in the future easily. Furthermore a `NotImplementedError` will be raised for every not supported virtualization/provisioning method.